util/explain_analyze_helper.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE FUNCTION pganalyze.explain_analyze(query text, params text[], param_types text[], analyze_flags text[]) RETURNS text AS $$
+DECLARE
+  prepared_query text;
+  params_str text;
+  param_types_str text;
+  explain_prefix text;
+  explain_flag text;
+  result text;
+BEGIN
+  SET TRANSACTION READ ONLY;
+
+  PERFORM 1 FROM pg_roles WHERE (rolname = current_user AND rolsuper) OR (pg_has_role(oid, 'MEMBER') AND rolname IN ('rds_superuser', 'azure_pg_admin', 'cloudsqlsuperuser'));
+  IF FOUND THEN
+    RAISE EXCEPTION 'cannot run: EXPLAIN ANALYZE helper is owned by superuser - recreate function with lesser privileged user';
```

Comment:
Reviewing this error message in-app, I wonder if for consistency we instead say here and in the other messages:

```suggestion
    RAISE EXCEPTION 'cannot run: pganalyze.explain_analyze(...) function is owned by superuser - recreate function with lesser privileged user';
```

Before:
<img width="867" alt="Screenshot 2024-12-18 at 11 24 41 PM" src="https://github.com/user-attachments/assets/26226b6b-371b-4a39-84b9-ca181fe92862" />


After:
<img width="842" alt="Screenshot 2024-12-18 at 11 25 08 PM" src="https://github.com/user-attachments/assets/69dbdcdb-4dd5-4b00-a742-ddb884ed3500" />

Reply:
Is the `(...)` really necessary? I'd tend to think that saying "the pganalyze.explain_analyze function" is plenty clear. Not a big deal, just trying to reduce character count.

Reply:
Yeah, fair point, especially since we say function right after.